llvm-tools/src/main/scala/scala/scalanative/llvm/LLVM.scala

```diff
@@ -0,0 +1,158 @@
+package scala.scalanative
+package llvm
+
+import java.io.File
+
+import scala.sys.process.{Process, ProcessLogger}
+import scala.util.Try
+
+import System.{lineSeparator => nl}
+
+object LLVM {
```

Comment:
What's the vision behind this refactoring? What should go to `LLVM` rather than sbt plugin and why? This needs to be explained in doc comment.